### DISCUSSION TITLE
renderGroupCell not being triggered

### DISCUSSION THREAD

**Discussion by NickyAlfEdge** [Comcast/react-data-grid#3311] [Q&A]
Hi all, I have been building locally a solution using Vite and in doing so the renderGroupCell is working fine, however, when I move the solution over to using WebPack I am finding that the renderGroupCell column property is not being executed. 

A simple, ```console.log(props);``` within the renderGroupCell shows that it is being executed when running with Vite, however, the log is not being reached when running via WebPack.

Has anybody encountered anything similar to this?

**Comment by Yangba** [comment]
Have you tried groupFormatter? Works for me.
`groupFormatter:({ childRows }) => {
        return childRows
          .reduce((prev, {silver}) => prev + silver, 0)         
      }`
